concordia/static/scss/base.scss

```diff
@@ -613,6 +613,7 @@ ul.nav-secondary {
 
 .view-homepage .carousel-indicators .active {
     background: url('data:image/svg+xml;charset=utf8,%3Csvg%20height%3D%2212%22%20viewBox%3D%220%200%2012%2012%22%20width%3D%2212%22%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%3E%3Ccircle%20cx%3D%226%22%20cy%3D%226%22%20fill%3D%22%23F05129%22%20fill-rule%3D%22evenodd%22%20r%3D%225.5%22%20stroke%3D%22%23fff%22/%3E%3C/svg%3E');
+    background-repeat: no-repeat;
```

Comment:
This shouldn't have been necessary. I wonder what changed to break the consistent size for the indicators — it's in master but not release currently.

Reply:
I was wondering the same thing.

Reply:
Looks like it was a change added in Bootstrap 4.2 to improve usability by making the click targets taller with a 10px transparent border: https://github.com/twbs/bootstrap/pull/26996

We used 4.1.3 until https://github.com/LibraryOfCongress/concordia/commit/5f9efa090984022b120c2117b95f905b08121f3b